streams/core.py

```diff
@@ -524,17 +530,23 @@ def update(self, x, who=None):
 
 
 class combine_latest(Stream):
-    def __init__(self, *children):
+    def __init__(self, *children, emit_on=None):
```

Comment:
Right, you might have to do `**kwargs` and then `emit_on = kwargs.pop('emit_on', None)`